c10/cuda/CUDACachingAllocator.cpp

```diff
@@ -190,13 +195,16 @@ struct Block {
   BlockPool* pool{nullptr}; // owning memory pool
   void* ptr{nullptr}; // memory address
   bool allocated{false}; // in-use flag
+  bool mapped{true}; // is memory in expandable_segment_
+                     // present, only false when allocated is also false.
```

Comment:
nit: `present`/`non-null`, also it's surprising to me that the default here is `true` but also the default for `expandable_segment_` is nullptr

Reply:
Clarified inline: this indicates that the Block has physical memory mapped to it.  When not using expandable segments, Blocks always are backed by real memory so this ends up being true.